src/vulkan-renderer/wrapper/instance.cpp

```diff
@@ -41,6 +41,12 @@ Instance::Instance(const std::string &application_name, const std::string &engin
 #endif
     };
 
+    spdlog::debug("Initializing glfw library.");
+
+    if (!glfwInit()) {
```

Comment:
Should GLFW init code be part of the instance? I think it should either be part of `wrapper::Window` or in a new `wrapper::GLFWContext` class.

Reply:
Right. I thought about this as well. I will move it to a separate `GLFWContext` class.

Reply:
Done.